ci-wheel.Dockerfile

```diff
@@ -71,14 +74,17 @@ RUN case "${LINUX_VER}" in \
       ;; \
     "rockylinux"*) \
         dnf update -y \
+        && dnf install -y epel-release \
+        && dnf update -y \
         && dnf install -y \
-          epel-release which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
+          which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
           sqlite-devel xz xz-devel libffi-devel curl git ncurses-devel numactl \
-          numactl-devel openssh-clients libcudnn8-devel zip jq \
+          numactl-devel openssh-clients libcudnn8-devel zip jq openslide-devel \
           protobuf-compiler autoconf automake libtool dnf-plugins-core cmake \
         && dnf config-manager --set-enabled powertools \
         && dnf install -y blas-devel lapack-devel \
         && dnf -y install gcc-toolset-11-gcc gcc-toolset-11-gcc-c++ \
+        && dnf -y yasm \
```

Comment:
```suggestion
        && dnf install -y yasm \
```

Reply:
Done in commit: b77c613eb92dc42d0600817d504b2797cbfbba8d